<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointXYZ</class>
 <widget class="QWidget" name="PointXYZ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>71</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>编辑坐标</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0,1,0,1">
     <item>
      <widget class="QLabel" name="titleX">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="posX">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleY">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="posY">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="titleZ">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="posZ">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> mm</string>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="axisRotationTitle">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="posRotation">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string> °</string>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="endPointFromTable">
       <property name="minimumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>列表取点</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="toEndpoint">
       <property name="minimumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>移到坐标</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getEndpoint">
       <property name="minimumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>63</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>取坐标</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>posX</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PointXYZ</receiver>
   <slot>onValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posY</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PointXYZ</receiver>
   <slot>onValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>posZ</sender>
   <signal>valueChanged(double)</signal>
   <receiver>PointXYZ</receiver>
   <slot>onValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onValueChanged()</slot>
 </slots>
</ui>
